psq/queue.py

```diff
@@ -64,7 +69,11 @@ def _get_or_create_subscription(self):
         if not subscription.exists():
```

Comment:
not sure, but it seems if you're wrapping in a try catch there's not a huge purpose to the first if. As Danny would say IBTAFTP . Plus you save the round trip of the exists check.